coalib/bearlib/abstractions/Linter.py

```diff
@@ -115,6 +142,9 @@ def _prepare_options(options):
 
 
 def _create_linter(klass, options):
+
+    _prepare_options(options, klass.__name__)
```

Comment:
please provide the class type itself instead of the name, then we have complete access to the bear in `_prepare_options`, which can be potentially used later.